### PR TITLE
Update test plan for access request, mfa, switchback banner

### PR DESCRIPTION
**Pull request by kimlisa**
#### Description
Update test plan with new UI access request feature and mfa

edit: added switchback test

### CODE REVIEW

File: docs/testplan.md [hunk 1]
```diff
@@ -490,6 +510,26 @@ With the previous role you created from `Strategy Reason`, change `request_acces
 - [ ] Verify that account is locked after several unsuccessful attempts
 - [ ] Verify that redirect to original URL works after successful login
 
+
+## Multi-factor Authentication (mfa)
+Create/modify `teleport.yaml` and set the following authentication settings under `auth_service`
+```
+authentication:
+  type: local
+  second_factor: optional
```
Comment (awly) [inline docs/testplan.md:519]: You also have to configure the `u2f` section for this: https://goteleport.com/docs/access-controls/guides/u2f/#enable-u2f-support

**Review by awly — APPROVED** [review]
LGTM for MFA checks

**Review by russjones — APPROVED** [review]
Bot.

File: docs/testplan.md (hunk 1 above)
Comment (kimlisa) [reply docs/testplan.md:523]: added

**Pull request merged**